src/packages/sdk/src/engine-commands/getDmmf.ts

```diff
@@ -70,6 +70,8 @@ async function getDmmfBinary(options: GetDMMFOptions): Promise<DMMF.Document> {
     EngineTypes.queryEngine,
     options.prismaPath,
   )
+  debug(`Using Query Engine Binary at: ${queryEnginePath}`)
```

Comment:
Are these debug logs useful, they seem redundant, can we remove them? @williamluke4

Reply:
Ok they are not redundant, It's just me and GitHub UI 👁️